### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by ayshsandu**

### CODE REVIEW

File: en/docs/learn/configuring-saml2-web-single-sign-on.md [hunk 1]
```diff
@@ -237,7 +237,7 @@ To configure SAML2 Web SSO:
...
-    <p>In Travelocity sample SP, this value can be set by modifying SAML2.IdPEntityId value mentioned in the <a href="http://travelocity.com/WEB-INF/classes/travelocity.properties">travelocity.com/WEB-INF/classes/travelocity.properties</a> file, so that it reflects the value of the IdP Entity ID Alias you define in the SAML SP configuration.</p>
+    <p>In Travelocity sample SP, this value can be set by modifying SAML2.IdPEntityId value mentioned in the `<travelocity.com-hpme>/WEB-INF/classes/travelocity.properties` file, so that it reflects the value of the IdP Entity ID Alias you define in the SAML SP configuration.</p>
```
Comment (gomathyk) [inline en/docs/learn/configuring-saml2-web-single-sign-on.md]: Is this supposed to be <travelocity.com-home>?

File: en/docs/learn/querying-saml-assertions.md [hunk 2]
```diff
@@ -204,8 +204,7 @@ AssertionID for the travelocity sample application.
     and a response similiar to the following:
 
     ??? note "AssertionIDRequest Request"
-
-            ``` xml
+        ```xml
```
Comment (gomathyk) [inline en/docs/learn/querying-saml-assertions.md]: shall we fix this alignment?

File: en/docs/learn/querying-saml-assertions.md [hunk 3]
```diff
@@ -233,28 +232,54 @@ AssertionID for the travelocity sample application.
...
-            ```
+        ```
 
     ??? note "AssertionIDRequest Response"
-
-            ``` xml
-                <?xml version="1.0" encoding="UTF-8" ?>
-                <saml2p:Response xmlns:saml2p="urn:oasis:names:tc:SAML:2.0:protocol" ID="_714cddb7f1c42d64376f0e6bd9d2f310" IssueInstant="2016-09-12T03:18:31.233Z" Version="2.0">
-                    <saml2:Issuer xmlns:saml2="urn:oasis:names:tc:SAML:2.0:assertion" Format="urn:oasis:names:tc:SAML:2.0:nameid-format:entity">localhost</saml2:Issuer>
+        ```xml
```
Comment (gomathyk) [inline en/docs/learn/querying-saml-assertions.md]: shall we fix the alignment?

File: en/docs/learn/multi-factor-authentication-for-wso2-is-management-console.md [hunk 4]
```diff
@@ -10,6 +10,16 @@
     2.  **Assertion Consumer URLs**: `https://localhost:9443/acs`
     3.  **Enable Response Signing**: `true`
 
+        !!! Tip 
+           
+            When configuring single-sign-on for Management Console, "Issuer" value of the SAML request is default to **carbonServer** as mentioned above.
```
Comment (gomathyk) [inline en/docs/learn/multi-factor-authentication-for-wso2-is-management-console.md]: When configuring single-sign-on for the Management Console, the "Issuer" value of the SAML request is defaulted to **carbonServer** as mentioned above.

File: en/docs/learn/multi-factor-authentication-for-wso2-is-management-console.md [hunk 5]
```diff
@@ -10,6 +10,16 @@
...
+            You can change this value by adding following configuration to `<IS_HOME>/repository/conf/deployment.toml` file.
```
Comment (gomathyk) [inline en/docs/learn/multi-factor-authentication-for-wso2-is-management-console.md]: by adding the following configuration to the `<IS_HOME>/repository/conf/deployment.toml` file.

File: en/docs/learn/configuring-saml2-web-single-sign-on.md (hunk 1 above)
Comment (ayshsandu) [reply en/docs/learn/configuring-saml2-web-single-sign-on.md]: fixed

**Pull request merged**